verilog/formatting/formatter_test.cc

```diff
@@ -9347,7 +9347,7 @@ static constexpr FormatterTestCase kFormatterTestCasesEnumDeclarations[] = {
      "  kA   = 1,  // value kA\n"
      "  // hello world\n"
      "  kAB  = 2,  // value kAB\n"
-     "  kABC = 3  // value kABC"
+     "  kABC = 3   // value kABC"
```

Comment:
Maybe also add a test case with no comment in between? I mean like:
```
  kAB  = 2, // comment
  kABC = 3,
  kABCD // comment
```

Reply:
I added above test.